samples/graph-change-notification/nodejs/README.md

```diff
@@ -0,0 +1,88 @@
+﻿### Change Notification sample
+
+Bot Framework v4 ChangeNotification sample.
+
+This sample app demonstrates sending change notifications to user presence in Teams based on user presence status.
+
+
+## Prerequisites
+
+- Microsoft Teams is installed and you have an account
+- [.NET Core SDK](https://dotnet.microsoft.com/download) version 3.1
```

Comment:
Could you please remove this and update steps specific to node.js enviroment?

Reply:
Yes